### PR TITLE
Fix typos in libgnucash/

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L ans,ba,parm,parms,numer`

### CODE REVIEW

File: libgnucash/doc/finutil.html [hunk 1]
```diff
@@ -696,7 +696,7 @@ <h1>Financial Calculator Usage</h1></a>
 [Hh](elp) to Display This Help
 [Qq](uit)? to Quit
 [Ss](tatus)? to Display Status of Computations
-[Uu](ser) Display User Defined Variables
```
Comment (fellen) [inline libgnucash/doc/finutil.html:699]: wrong

File: libgnucash/doc/finutil.html [hunk 2]
```diff
@@ -756,7 +756,7 @@ <h2>Calculator Commands</h2></a>
 Initial Payment Date: Thu Aug 01 00:00:00 1996(2450297)
 <>
 </pre>
-<li>[Uu](ser) Display User Defined Variables
+<li>[Uu](set) Display User Defined Variables
```
Comment (fellen) [inline libgnucash/doc/finutil.html:759]: too

**Review by fellen — CHANGES_REQUESTED** [review]
I found 2 wrong interpreted strings, but perhaps a native speaker should review the end 's'es.

File: libgnucash/doc/finutil.html (hunk 1 above)
Comment (luzpaz) [reply libgnucash/doc/finutil.html:699]: Sorry about that. Will revert.

**Comment by fellen** [comment]
Applied after reverting the two undesired changes.

Thank you!

**Pull request closed without merging**